game_util/pet_config.py

```diff
@@ -13,7 +13,7 @@ class PetConfig:
     FONT = '../my_pet/theme_items/Starborn.ttf'
 
     # Colors
-    BLACK = (0, 0, 0)
+    BLACK = (0, 0, 5)
```

Comment:
I would say give this a different Name like `SPRITE_BG` 🤔 cause that way we would not confuse that with actual black. And then revert black back to `(0, 0, 0)`

Reply:
Sounds good